src/lib/Zikula/Core/Controller/AbstractController.php

```diff
@@ -7,141 +17,209 @@
 use Symfony\Component\HttpFoundation\StreamedResponse;
 use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
 use Symfony\Component\Security\Core\Exception\AccessDeniedException;
-use Zikula\Common\I18n\Translator;
-use Zikula\Common\I18n\TranslatorAwareInterface;
 use Zikula\Core\AbstractBundle;
 
-abstract class AbstractController extends Controller implements TranslatorAwareInterface
+abstract class AbstractController extends Controller
```

Comment:
-use Zikula\Common\I18n\Translator;
-use Zikula\Common\I18n\TranslatorAwareInterface;

Both classes are not used and there is no new TranslatorInterface at the moment.